pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -294,48 +305,51 @@ func ContainerFromNative(n *native.Container) (*Container, error) {
 	return c, nil
 }
 
-func ImageFromNative(n *native.Image) (*Image, error) {
-	i := &Image{}
-
-	imgoci := n.ImageConfig
+func ImageFromNative(nativeImage *native.Image) (*Image, error) {
```

Comment:
Light cleanup:
- improve variable name readability
- introduce variant and virtual size
- populate repodigest